### ISSUE TITLE
How to use shumway and pixi to simulate flash api

### ISSUE THREAD

**Issue by lovoror** [ivanpopelyshev/shumway#4]
Any suggestion?

**Comment by ivanpopelyshev** [comment]
Yes, its possible, you have to wait when I publish all my stuff :) At this point I can say that almost all flash features are working, including filters and multiline TextField edits with ctrl+c ctrl+v. But, you have to wait two or three month :(

Alternatively, there's a guy on html5gamedevs forum that has his own bindings for pixi for converting flash games. 

Alternatively, look at AwayFL (awayjs) , they published their results.

**Comment by lovoror** [comment]
Great job!
Thanks, But I can't wait for it  so much long time.
I will do it by myself.
Thanks again!

**Comment by ivanpopelyshev** [comment]
You can ask here if you have troubles with any parts of flash graphics API, i know many tricks :)
